flipperdevices/Flipper-Android-App
# Fix/infrared remote placeholders design

**Background**

Previously flipper android app had two ui states `ErrorContent` and `ComposableInfraredRemotes`. This PR intends to add more UI states which will help user to understand current `EmulatorState` more precisely

**Changes**

- Add fade modifier
- Fix placeholder color
- Fix edit remote save button color
- Add syncing ui state display
- Add empty remote buttons hint
- Improve multiple buttons display upon flipper not connected 

**Test plan**

- Open infrared key info
- Disconnect flipper device to see disconnected state with no remote buttons
- Disconnect flipper device to see disconnected state with 1-2 remote buttons
- Disconnect flipper device to see disconnected state with >3 remote buttons
- Emulate low connection to see connecting state
- Connect flipper to see connected state with remote buttons
- Connect flipper to see connected state without remote buttons
